src/app/organization/tellers/view-cashier/view-cashier.component.html

```diff
@@ -0,0 +1,71 @@
+<div fxLayout="row" fxLayoutAlign="end" fxLayoutGap="2%" fxLayout.lt-md="column" class="container m-b-20">
+  <button mat-raised-button color="primary" [routerLink]="['edit']">
```

Comment:
dont add routerlink until component is ready

Reply:
ok